rego/data.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
How/where is this file used?

Reply:
It's just the mockDB from your program. Maybe we don't need it, I will remove it.